core/src/main/resources/lib/audio.fjs

```diff
@@ -98,7 +98,8 @@ export default (componentType: ComponentType) => ({ ophan, dom, viewport }: Serv
   };
 
   const showProgress = (audio: Audio, percentPlayed: Number, now: Number, played: string) => {
-    const gradientDescription = `linear-gradient(to right, #C70000 ${percentPlayed}%, #afafaf ${percentPlayed}%)`;
+    // TODO: make the input control's background honour the pillar colour - meanwhile use the 'news' default
+    const gradientDescription = `linear-gradient(to right, #e00000 ${percentPlayed}%, #afafaf ${percentPlayed}%)`;
```

Comment:
This will effectively override the definition you set in the CSS. What I suggest instead is that you parse it on start, e.g. assuming a `progressBar` reference to the DOM element:
```js
const styles = getComputedStyle(progressBar);
const defaultValue = '#c70000';
const colorRegex = /##[\w\d]{6}/;
const pillarColour = (styles.getPropertyValue('background') || defaultValue).match(colorRegex);
// pillarColour[0] has the colour.
```
With CSS variables, this will be way easier:
```js
const styles = getComputedStyles(document.body);
const pillarColour = styles.getPropertyValue('--c-main') || defaultValue;
````